projects/vdk-control-cli/src/vdk/internal/control/command_groups/login_group/login.py

```diff
@@ -112,17 +110,25 @@ def login(
     oauth2_exchange_url,
 ):
     if auth_type == LoginTypes.CREDENTIALS.value:
-        if not client_id or not oauth2_discovery_url or not oauth2_exchange_url:
-            click.echo(
-                f"Login type: {auth_type} requires arguments:, --client-secret, --client-id, "
-                f"--oauth2-exchange-url, --oauth2-discovery-url"
+        try:
+            auth = Authentication(
+                client_id=client_id,
+                client_secret=client_secret,
+                auth_discovery_url=oauth2_discovery_url,
+                authorization_url=oauth2_exchange_url,
+                auth_type=auth_type,
+                cache_locally=True,
             )
-        else:
-            redirect_auth = RedirectAuthentication(
-                client_id, client_secret, oauth2_discovery_url, oauth2_exchange_url
-            )
-            redirect_auth.authentication_process()
+            auth.authenticate()
             click.echo("Login Successful")
+        except VDKAuthException as e:
+            if "requires client_id and auth_discovery_url to be specified" in e.message:
```

Comment:
looking for test coverage this exception+message is raised in case of `not client_id or not oauth2_discovery_url or not oauth2_exchange_url`

Reply:
Good point. Test added. Thanks